app/assets/stylesheets/_card.scss

```diff
@@ -31,17 +31,46 @@
     }
   }
 
+  &.project-item{
+    display: flex;
+    flex-direction: column;
+  }
+
   .panel-heading {
     background-color: inherit;
     font-weight: bold;
     color: $darkgrey-2;
     border-bottom: 1px solid $lightgrey-11;
 
+    &.card-heading{
+      -webkit-box-align: center;
+      align-items: center;
+      -webkit-box-pack: justify;
+      justify-content: space-between;
+      display: -webkit-box;
+      display: flex;
+      flex-basis: 53px;
+    }
+
     .card-title {
       color: $darkgrey-3;
       text-decoration: none;
     }
 
+    .card-tag{
+      padding: 0 5px;
+      font-size: 9px;
+      border-radius: 5px;
+      color: #ffffff;
```

Comment:
Using this variable: https://github.com/Codeminer42/cm42-central/blob/master/app/assets/stylesheets/_variables.scss#L1

If there's some of color you want to use that is not in this file, you can add it.

The syntax to using variables in SCSS is:

```scss
color: $white;
```